doc/Menus.md

```diff
@@ -46,12 +46,16 @@ admin.menu(nga.menu()
 );
 ```
 
-*Tip*: `admin.menu()` is both a setter and a getter. You can modify an existing menu in the admin configuration by using `admin.menu().getChildByTitle()`
+By default parent menus will automatically close when none of their children are active. This can be deactivated with:
+```
+nga.menu().autoClose(false);
+```js
```

Comment:
bad formatting: the 'js' must be on the opening line. Also, please add a newline after the code snippet.